server/apps/publish/archive_publish.py

```diff
@@ -385,6 +377,80 @@ def publish(self, doc, updates, target_media_type=None):
 
         return queued
 
+    def sending_to_digital_subscribers(self, doc):
+        subscribers, subscribers_yet_to_receive = self.get_subscribers(doc)
+        filtered_subscribers = self.filter_subscribers(doc, subscribers, DIGITAL)
+        return len(filtered_subscribers) > 0
+
+    def get_subscribers(self, doc):
+        subscribers, subscribers_yet_to_receive = [], []
+        if self.published_state == 'corrected':
+            query = {'$and': [{'item_id': doc[config.ID_FIELD]}, {'publishing_action': 'published'}]}
```

Comment:
include `{'publishing_action': {'$in': ['published', 'correction']}}` for correction of a correction.